environment_setup/setup_software_arch.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# UBC Thunderbots Arch Linux Software Setup
+#
+# This script must be run with sudo! root permissions are required to install
+# packages and copy files to the /etc/udev/rules.d directory. The reason that the script
+# must be run with sudo rather than the individual commands using sudo, is that
+# when running CI within Docker, the sudo command does not exist since
+# everything is automatically run as root.
+#
+# This script will install all the required libraries and dependencies to build
+# and run the Thunderbots codebase. This includes being able to run the ai and
+# unit tests
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+
```

Comment:
Could we please put a big:
"WARNING: THIS IS NOT AN OFFICIALLY SUPPORTED SCRIPT. IT IS NOT RUN IN CI AND HAS NO GUARANTEE OF WORKING" warning here? Preferably in a big ASCII box.